### PR TITLE
Removing invalid adapter code

### PR DESCRIPTION
**Pull request by shlomisut**
Adding build command for jenkind, currently only with criteo

## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by shlomisut** [comment]
@Ginbar1 @borisjaskerovich @royasoolin  this is the first PR for prebid 1.x
comments inline

**Comment by ghost** [comment]
Build Passed

File: modules/appnexusBidAdapter.js [hunk 1]
```diff
@@ -30,7 +30,10 @@ const SOURCE = 'pbjs';
 
 export const spec = {
   code: BIDDER_CODE,
-  aliases: ['appnexusAst', 'brealtime', 'emxdigital', 'pagescience', 'defymedia', 'gourmetads', 'matomy', 'featureforward', 'oftmedia', 'districtm'],
+  aliases: ['appnexusAst', 'brealtime', 'emxdigital', 'pagescience', 'defymedia', 'gourmetads', 'matomy', 'featureforward', 'oftmedia', 'districtm'
```
Comment (shlomisut) [inline modules/appnexusBidAdapter.js:33]: aliases that were used in our system for multiple appnexus adapters

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "src/prebid.js",
   "scripts": {
     "test": "gulp test",
-    "lint": "gulp lint"
+    "lint": "gulp lint",
+    "build_prebid_js": "gulp build --modules=criteoBidAdapter"
```
Comment (shlomisut) [inline package.json:9]: this is what creates the prebid file we currently have
Add adapters to this list once you verified they work properly

**Comment by shlomisut** [comment]
Had to remove a9 adapters as otherwise the build would fail
Need to add `a9Parallel` to prebid 1.x too

**Pull request merged**